src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,66 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+
+            DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo);
+
+            Assert.NotNull(directorySecurity);
```

Comment:
Is there anything else we can validate here? e.g. that the right values are returned from properties like AccessRightType or anything else that has defaults we can verify?